tsx.mjs

```diff
@@ -1,7 +1,8 @@
+import fs from 'node:fs';
 import path from 'node:path';
 import { pathToFileURL } from 'node:url';
 
-import { transform } from 'esbuild';
+import { transformSync } from 'esbuild';
```

Comment:
You may want to try with SWC instead; Esbuild runs within a separate process, so there's some message passing overhead, especially when going sync.

Reply:
Mhmm. Thanks, I know though 🙂

Also https://github.com/JakobJingleheimer/nodejs-loaders/issues/10 ;)